polls/models.py

```diff
@@ -86,35 +87,17 @@ def __str__(self):
 
 
 class ExperimentType(models.Model):
-    type_choices = (
-        ("2AFC", "2AFC"),
-        ("A/B", "A/B"),
-    )
-    type = models.CharField(max_length=10, choices=type_choices)
+    type = models.CharField(max_length=100)
 
     def __str__(self):
         return f"{self.type}"
 
 
 class Experiment(models.Model):
-    name = models.CharField(max_length=50)
+    name = models.CharField(max_length=100)
     experiment_type = models.ForeignKey(ExperimentType, on_delete=models.CASCADE)
+    task_presentation = ArrayField(models.CharField(max_length=100), default=list)
+    annotation = ArrayField(models.CharField(max_length=100), default=list)
 
     def __str__(self):
         return f"{self.name}"
-
```

Comment:
I think we will be keeping these. We don't want ArrayField because it breaks sqlite support